app/src/main/res/values/strings.xml

```diff
@@ -247,8 +251,9 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="quest_generic_hasFeature_no">"No"</string>
     <string name="quest_busStopShelter_title">"Does this bus stop have a shelter?"</string>
     <string name="quest_bin_title">"Does this bus stop have a waste bin?"</string>
-    <string name="quest_busStopLit_title">"Is this bus stop well lit?"</string>
+    <string name="quest_busStopLit_title">"Is there lighting at this Bus stop?"</string>
     <string name="quest_busStopBench_title">"Does this bus stop have a bench?"</string>
+    <string name="quest_SidewalkLit_title">"How many lights do you see from the place you’re standing?"</string>
```

Comment:
Make sure to change this to a yes/no question (same as the street lit question, basically)